test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -367,7 +368,8 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		})
 		It("Should not have RPS configuration set when realtime workload hint is explicitly set", func() {
 
-			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil && !*profile.Spec.WorkloadHints.RealTime {
+			if profile.Spec.WorkloadHints != nil && profile.Spec.WorkloadHints.RealTime != nil &&
+				!*profile.Spec.WorkloadHints.RealTime && !profileutil.IsRpsEnabled(profile) {
```

Comment:
dito

Reply:
this is a check whether to remove all rps settings or not.